tests/lib/tools/os.query

```diff
@@ -45,6 +45,10 @@ is_focal() {
     grep -qFx 'UBUNTU_CODENAME=focal' /etc/os-release
 }
 
+is_ubuntu() {
+    grep -qFx 'ID=ubuntu' /etc/os-release || grep -qFx 'ID=ubuntu-core' /etc/os-release
```

Comment:
grep -qF "ID=ubuntu" would catch both but maybe it's better to be precise.

Reply:
I thought that but I think this way is more explicit about the different supported ID, does it make sense to keep it that way?